pyproject.toml

```diff
@@ -12,12 +12,11 @@ classifiers = [
     "License :: Public Domain",
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.5",
-    "Programming Language :: Python :: 3.6",
```

Comment:
Yes but the users of that operating system can get LTS from RedHat.

Reply:
But they cannot get latest web.py working on py3.6. Since no extra work introduced for py3.6 support, i hope we can keep it in `pyproject.toml`.

Reply:
`python3.6 -m pip install web.py==0.51`

Reply:
web.py 0.62 still works with py3.6, i just hope we can run next release with py3.6.

Let me repeat my opinion: since NO EXTRA WORK introduced to support py3.6, please let's keep it in `pyproject.toml` to benefit users running old but LTS OS release, e.g. RHEL/CentOS 7. When there's breaking changes which don't support py3.6, it's ok to drop py3.6 support then.

Reply:
Done.

Reply:
Thank you @cclauss :)